apps/service_orchestrate/lib/orchestrate/schedule/store.ex

```diff
@@ -6,14 +6,14 @@ defmodule Orchestrate.Schedule.Store do
 
   @spec persist(Schedule.t()) :: :ok
   def persist(%Schedule{} = schedule) do
-    Brook.ViewState.merge(@collection, identifier(schedule), %{schedule: schedule})
+    Brook.ViewState.merge(@collection, identifier(schedule), %{"schedule" => schedule})
```

Comment:
Why'd you make this change? Just looking to understand.

Reply:
Woops, I guess we didn't need to change this line. We had some integration tests fail when we called `Map.get` below this because calls to redis were returning string keys. I'm assuming because they are getting encoded/decoded?

I think we did this somewhere else, too. I will fix that

Reply:
No it's fine. I was just curious. Other services have a `store.ex` file with basically the same function calls. How do you feel about changing them there too? I'd like to be consistent.

Reply:
Sure, np